### PR TITLE
Update module name to match the repository.

### PR DESCRIPTION
**Pull request by creachadair**
Since this module was moved to a new repository, the go tool complains on
import that its declared name doesn't match its requirement spec:

```
 module declares its path as: github.com/interchainio/tm-load-test
        but was required as: github.com/informalsystems/tm-load-test
```

This updates the module declaration and the internal import paths, along with
the build string injection rule in the Makefile.

Since the repository was forwarded, and all the existing issues still work at
the new address, I also updated the changelog for consistency.

(I ran into this while investigating tendermint/tendermint/issues/7032)

### CODE REVIEW

**Review by thanethomson — APPROVED** [review]
Awesome, thanks for this @creachadair!

This repo does need a little TLC at the moment.

**Comment by creachadair** [comment]
Thanks for the review! I think you'll have to merge this, if you want to, since I do not have commit access here.

**Comment by creachadair** [comment]
> Ah, I see our repo's configured to only accept signed commits. It won't let me merge.

Blrgh. I need to go re-figure-out how to do signed commits. I used to have to do this, but it's been a while.

**Comment by thanethomson** [comment]
Don't worry, I can redo your changes in a separate PR - it's standard practice for us nowadays.

**Comment by thanethomson** [comment]
> In that case, may I go ahead and close this without disrupting your intent?

Done with #111. Thanks for bringing this to my attention!

**Pull request closed without merging**
